xml/operations-configuring-identity-websso.xml

```diff
@@ -402,16 +405,10 @@ keystone_sp_conf:
   <orderedlist>
    <listitem>
     <para>
-     In the preceding config file, /tmp/adfs_config.yml, make sure the
-     idp_metadata_file references the previously generated ADFS metadata file.
-     In this case:
-    </para>
-<screen>idp_metadata_file: /tmp/adfs_metadata.xml</screen>
-   </listitem>
-   <listitem>
-    <para>
-     Create a mapping file that is referenced from the preceding config file,
-     such as /tmp/adfs_sp_mapping.json. rules_file: /tmp/adfs_sp_mapping.json.
+     Create a mapping file, adfs_mapping.json, that is referenced from the
+     preceding config file in
+     <filename>/var/lib/ardana/openstack/my_cloud/config/keystone/</filename>.
+     rules_file: /var/lib/ardana/openstack/my_cloud/config/keystone/adfs_mapping.json.
```

Comment:
What's this second line mean?

Reply:
That's an example of the rules_file line. Maybe I need to encapsulate it in <screen>?

Reply:
Yeah :) it would be helpful to be marked up.

Reply:
Done. Put it in <screen>.